exporter/dorisexporter/exporter_common.go

```diff
@@ -118,3 +119,19 @@ func createAndUseDatabase(ctx context.Context, conn *sql.DB, cfg *Config) error
 	_, err = conn.ExecContext(ctx, "USE "+cfg.Database)
 	return err
 }
+
+type metric interface {
+	dMetricGauge | dMetricSum | dMetricHistogram | dMetricExponentialHistogram | dMetricSummary
+}
+
+func toJsonLines[T dLog | dTrace | metric](data []*T) ([]byte, error) {
+	buf := &bytes.Buffer{}
+	enc := json.NewEncoder(buf)
+	for _, d := range data {
+		err := enc.Encode(d)
```

Comment:
did you need a `\n` character in between the objects?

Reply:
Here [TestToJsonLines](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36912/files#diff-3206ee72b430e624c8abc8552e6c0b8cc970def18c5b8e6c940f8f0eb651bac7).